nanotune/tuningstages/base_tasks.py

```diff
@@ -3,9 +3,11 @@
 # This software is released under the MIT License.
 # https://opensource.org/licenses/MIT
 import copy
+from dataclasses import asdict
 import datetime
 import json
 import logging
+from os import initgroups
```

Comment:
This is only available on linux and perhaps mac. should this not work on windows

Reply:
Not sure how this got in. It is not used and now also removed.